lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -90,6 +91,8 @@ def record_batch txn, logs
       end
 
       def create_event priority, formatted_message, severity
+        formatted_message = truncate_message(formatted_message) if formatted_message.bytesize > MAX_BYTES
```

Comment:
I think we should drop the conditional aspect (`if formatted_message.bytesize > MAX_BYTES`) here. The new `truncate_message` method performs a very similar check. As things are currently, we are always performing one check and sometimes two. If we get rid of this conditional here, we'll only ever be performing one check.

We could also have `truncate_message` be destructive and then just have this one line be `truncate_message(formatted_message)`.

Reply:
Thanks for noticing this! I meant to get rid of one of these based on a performance test before I pushed this up. I haven't run said test yet. I will and remove whichever one is slower. 

If they're about the same, do you have a preference for one over the other?

Reply:
I think `truncate_message` should be the one to do the check.